src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -573,6 +570,25 @@ const generateTaskRole = (frameworkName, taskRole, jobInfo, frameworkEnvList, co
       name: `${encodeName(frameworkName)}-regcred`,
     });
   }
+  // add storages
```

Comment:
what if `teamwise_storage` in runtime plugin? Can it work?

Reply:
if old config `teamwise_storage` is used, runtime plugin can mount the storage as long as the plugin hasn't been deleted
if new config is used, rest server will add pod spec and runtime plugin can be skipped

Reply:
But I saw you remove `storage_config` when generate task_role. Runtime need this config, and need to read k8s secret to get storage info. After this change, seems `teamwise_plugin`will not work. (We don't have `storage secret` in k8s, and runtime can't get config through ENV)

Reply:
updated, `teamwise_storage` config will be converted to `storages` config